src/components/ResourceItem/ResourceItem.scss

```diff
@@ -318,19 +323,6 @@ $resource-list-item-variables: (
 }
 
 .newDesignLanguage {
-  @include focus-ring;
-
-  // stylelint-disable selector-max-specificity
-  &.focused:not(:active):not(.focusedInner) {
-    @include breakpoint-after(resource-list-item(breakpoint-small)) {
-      @include focus-ring($style: 'focused');
-    }
-
-    @include breakpoint-before(resource-list-item(breakpoint-small), false) {
-      box-shadow: inset 0.2rem 0 0 var(--p-focused);
-    }
```

Comment:
Did the ndl mocks remove this? iirc we wanted to keep the left onset border on small screens

Reply:
That makes sense, let me make sure this is good.